training/training.module

```diff
@@ -0,0 +1,378 @@
+<?php
+
+/**
+ * Implements hook_menu
+ * @return array
+ */
+function training_menu(){
+  $menu = array();
+  $menu['training/menu'] = array(
+    'title' => 'Training Landing Page',
+    'description' => 'This page is Landing Page',
+    'page callback' => 'training_page_html',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+    'expanded' => true,
+  );
+  $menu['training/menu/custom_access '] = array(
+    'title' => 'Custom Access Menu Item',
+    'description' => 'This page is accessible to authenticated users only',
+    'page callback' => 'training_page_html',
+    'access callback' => 'training_custom_access',
+    'access arguments' => array('authenticated user'),
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/menu/permissioned '] = array(
+    'title' => 'Permissioned Menu Item',
+    'description' => 'This page is accessible to permissioned users only',
+    'page callback' => 'training_page_html',
+    'access arguments' => array('train', ),
```

Comment:
array('train', ) - either remove trailing comma or add the second element to the array